astrokat/observe_main.py

```diff
@@ -38,14 +38,40 @@
 DUMP_RATE_TOLERANCE = 0.002
 
 
+# -- Utility functions --
 def __horizontal_coordinates__(target, observer, datetime_):
     """Utility function to calculate ephem horizontal coordinates"""
     observer.date = ephem.date(datetime_)
     target.compute(observer)
     return target.az, target.alt
 
 
-def observe(session, target_info, **kwargs):
+def __same_day__(start_lst, end_lst, local_lst):
```

Comment:
I see we had some of these before, but why do these functions have leading and trailing `__` ?  That is normally used for magic methods in Python objects.  For "private" functions a single leading underscore is normally used.

Reply:
@bngcebetsha Please update the names of these functions.